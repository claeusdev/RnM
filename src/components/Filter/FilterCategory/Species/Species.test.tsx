import { render } from "@testing-library/react";
import SpeciesFilter from "./Species";

test("renders character location/status", () => {
  const component = render(<SpeciesFilter updateSpecies={jest.fn()} updatePageNumber={jest.fn()}/>);
  expect(component).toMatchSnapshot();
});
