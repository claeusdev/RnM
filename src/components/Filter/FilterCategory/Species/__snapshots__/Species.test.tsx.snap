// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders character location/status 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="accordion-item "
      >
        <h2
          class="accordion-header"
          id="headingTwo"
        >
          <button
            aria-controls="collapseTwo"
            aria-expanded="false"
            class="accordion-button collapsed"
            data-bs-target="#collapseTwo"
            data-bs-toggle="collapse"
            type="button"
          >
            Species
          </button>
        </h2>
        <div
          aria-labelledby="headingTwo"
          class="accordion-collapse collapse"
          data-bs-parent="#accordionExample"
          id="collapseTwo"
        >
          <div
            class="accordion-body d-flex flex-wrap gap-3"
          >
            <style>
              
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
            </style>
            <div
              class="form-check"
            >
              <input
                class="form-check-input x"
                id="species-0"
                name="species"
                type="radio"
              />
              <label
                class="btn btn-outline-primary"
                for="species-0"
              >
                Human
              </label>
            </div>
            <style>
              
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
            </style>
            <div
              class="form-check"
            >
              <input
                class="form-check-input x"
                id="species-1"
                name="species"
                type="radio"
              />
              <label
                class="btn btn-outline-primary"
                for="species-1"
              >
                Alien
              </label>
            </div>
            <style>
              
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
            </style>
            <div
              class="form-check"
            >
              <input
                class="form-check-input x"
                id="species-2"
                name="species"
                type="radio"
              />
              <label
                class="btn btn-outline-primary"
                for="species-2"
              >
                Humanoid
              </label>
            </div>
            <style>
              
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
            </style>
            <div
              class="form-check"
            >
              <input
                class="form-check-input x"
                id="species-3"
                name="species"
                type="radio"
              />
              <label
                class="btn btn-outline-primary"
                for="species-3"
              >
                Poopybutthole
              </label>
            </div>
            <style>
              
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
            </style>
            <div
              class="form-check"
            >
              <input
                class="form-check-input x"
                id="species-4"
                name="species"
                type="radio"
              />
              <label
                class="btn btn-outline-primary"
                for="species-4"
              >
                Mythological
              </label>
            </div>
            <style>
              
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
            </style>
            <div
              class="form-check"
            >
              <input
                class="form-check-input x"
                id="species-5"
                name="species"
                type="radio"
              />
              <label
                class="btn btn-outline-primary"
                for="species-5"
              >
                Unknown
              </label>
            </div>
            <style>
              
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
            </style>
            <div
              class="form-check"
            >
              <input
                class="form-check-input x"
                id="species-6"
                name="species"
                type="radio"
              />
              <label
                class="btn btn-outline-primary"
                for="species-6"
              >
                Animal
              </label>
            </div>
            <style>
              
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
            </style>
            <div
              class="form-check"
            >
              <input
                class="form-check-input x"
                id="species-7"
                name="species"
                type="radio"
              />
              <label
                class="btn btn-outline-primary"
                for="species-7"
              >
                Disease
              </label>
            </div>
            <style>
              
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
            </style>
            <div
              class="form-check"
            >
              <input
                class="form-check-input x"
                id="species-8"
                name="species"
                type="radio"
              />
              <label
                class="btn btn-outline-primary"
                for="species-8"
              >
                Robot
              </label>
            </div>
            <style>
              
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
            </style>
            <div
              class="form-check"
            >
              <input
                class="form-check-input x"
                id="species-9"
                name="species"
                type="radio"
              />
              <label
                class="btn btn-outline-primary"
                for="species-9"
              >
                Cronenberg
              </label>
            </div>
            <style>
              
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
            </style>
            <div
              class="form-check"
            >
              <input
                class="form-check-input x"
                id="species-10"
                name="species"
                type="radio"
              />
              <label
                class="btn btn-outline-primary"
                for="species-10"
              >
                Planet
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="accordion-item "
    >
      <h2
        class="accordion-header"
        id="headingTwo"
      >
        <button
          aria-controls="collapseTwo"
          aria-expanded="false"
          class="accordion-button collapsed"
          data-bs-target="#collapseTwo"
          data-bs-toggle="collapse"
          type="button"
        >
          Species
        </button>
      </h2>
      <div
        aria-labelledby="headingTwo"
        class="accordion-collapse collapse"
        data-bs-parent="#accordionExample"
        id="collapseTwo"
      >
        <div
          class="accordion-body d-flex flex-wrap gap-3"
        >
          <style>
            
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
          </style>
          <div
            class="form-check"
          >
            <input
              class="form-check-input x"
              id="species-0"
              name="species"
              type="radio"
            />
            <label
              class="btn btn-outline-primary"
              for="species-0"
            >
              Human
            </label>
          </div>
          <style>
            
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
          </style>
          <div
            class="form-check"
          >
            <input
              class="form-check-input x"
              id="species-1"
              name="species"
              type="radio"
            />
            <label
              class="btn btn-outline-primary"
              for="species-1"
            >
              Alien
            </label>
          </div>
          <style>
            
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
          </style>
          <div
            class="form-check"
          >
            <input
              class="form-check-input x"
              id="species-2"
              name="species"
              type="radio"
            />
            <label
              class="btn btn-outline-primary"
              for="species-2"
            >
              Humanoid
            </label>
          </div>
          <style>
            
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
          </style>
          <div
            class="form-check"
          >
            <input
              class="form-check-input x"
              id="species-3"
              name="species"
              type="radio"
            />
            <label
              class="btn btn-outline-primary"
              for="species-3"
            >
              Poopybutthole
            </label>
          </div>
          <style>
            
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
          </style>
          <div
            class="form-check"
          >
            <input
              class="form-check-input x"
              id="species-4"
              name="species"
              type="radio"
            />
            <label
              class="btn btn-outline-primary"
              for="species-4"
            >
              Mythological
            </label>
          </div>
          <style>
            
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
          </style>
          <div
            class="form-check"
          >
            <input
              class="form-check-input x"
              id="species-5"
              name="species"
              type="radio"
            />
            <label
              class="btn btn-outline-primary"
              for="species-5"
            >
              Unknown
            </label>
          </div>
          <style>
            
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
          </style>
          <div
            class="form-check"
          >
            <input
              class="form-check-input x"
              id="species-6"
              name="species"
              type="radio"
            />
            <label
              class="btn btn-outline-primary"
              for="species-6"
            >
              Animal
            </label>
          </div>
          <style>
            
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
          </style>
          <div
            class="form-check"
          >
            <input
              class="form-check-input x"
              id="species-7"
              name="species"
              type="radio"
            />
            <label
              class="btn btn-outline-primary"
              for="species-7"
            >
              Disease
            </label>
          </div>
          <style>
            
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
          </style>
          <div
            class="form-check"
          >
            <input
              class="form-check-input x"
              id="species-8"
              name="species"
              type="radio"
            />
            <label
              class="btn btn-outline-primary"
              for="species-8"
            >
              Robot
            </label>
          </div>
          <style>
            
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
          </style>
          <div
            class="form-check"
          >
            <input
              class="form-check-input x"
              id="species-9"
              name="species"
              type="radio"
            />
            <label
              class="btn btn-outline-primary"
              for="species-9"
            >
              Cronenberg
            </label>
          </div>
          <style>
            
          .x:checked + label {
            background-color: #0b5ed7;
            color: white;
          }
          input[type="radio"] {
            display: none;
          }
        
          </style>
          <div
            class="form-check"
          >
            <input
              class="form-check-input x"
              id="species-10"
              name="species"
              type="radio"
            />
            <label
              class="btn btn-outline-primary"
              for="species-10"
            >
              Planet
            </label>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
